packages/moonstone/Input/InputSpotlightDecorator.js

```diff
@@ -155,11 +176,15 @@ const InputSpotlightDecorator = hoc((config, Wrapped) => {
 			if (focusChanged) {
 				if (this.state.focused === 'input') {
 					forward('onActivate', {type: 'onActivate'}, this.props);
-					lockPointer(this.state.node);
+					if (!unLockedPointer) {
+						lockPointer(this.state.node);
+					}
```

Comment:
If we have to wrap `lockPointer` and `releasePointer` with `unLockedPointer` condition, why don't you define new functions for them in which `unLockedPointer` condition is checked. Then we could avoid making a mistake to forget wrapping them with the condition.

Reply:
It's a good idea but  `if` condition is used only three times in total. I think this code will be refactoring next year when we are thinking of handling touchable events. I don't know which is better.

Reply:
OK. Let's consider it later.